code/modules/clothing/spacesuits/rig/suits/station.dm

```diff
@@ -162,6 +166,63 @@ Advanced Voidsuit: Technomancer Exultant
 	desc = "Advanced magnetic boots that have a lighter magnetic pull, placing less burden on the wearer."
 	mag_slow = 1
 
+/***************************************
+Technomancer RIG
+***************************************/
+/obj/item/weapon/rig/techno
+	name = "technomancer RIG suit control module"
+	suit_type = "technomancer RIG suit"
+	desc = "An advanced RIG suit that protects against hazardous, low pressure and high temperature environments."
+	icon_state = "techno_rig"
+	rarity_value = 20
+	armor = list(
+		melee = 40,
+		bullet = 40,
+		energy = 40,
+		bomb = 50,
+		bio = 100,
+		rad = 100
+	)
+	slowdown = 1
+	drain = 3
+	offline_slowdown = 3
+	offline_vision_restriction = 0
+
+	max_heat_protection_temperature = FIRESUIT_MAX_HEAT_PROTECTION_TEMPERATURE
+
+	helm_type = /obj/item/clothing/head/space/rig/techno
+	glove_type = /obj/item/clothing/gloves/rig/techno
+	boot_type = /obj/item/clothing/shoes/magboots/rig/techno
+
+	extra_allowed = list(
+		/obj/item/weapon/storage/toolbox,
+		/obj/item/weapon/storage/briefcase/inflatable,
+		/obj/item/device/t_scanner,
+		/obj/item/weapon/rcd
+	)
+
+	req_access = list(access_engine)
+	req_one_access = list()
+	spawn_blacklisted = TRUE
```

Comment:
Why is it blacklisted?

Reply:
For same reason TM voidsuit proper, mining vidsuit, IH RIG and others are.